src/data/radfaces.json

```diff
@@ -0,0 +1,526 @@
+  "last_name": "Pataki",
+  "image": "http://www.radfaces.com/images/avatars/helga-pataki.jpg",
+  "location": "Hillwood, WA"
+}, {
+  "first_name": "Laura",
+  "last_name": "Powers",
+  "image": "http://www.radfaces.com/images/avatars/laura-powers.jpg",
+  "location": "Springfield, IL"
+}, {
+  "first_name": "Lydia",
+  "last_name": "Deetz",
+  "image": "http://www.radfaces.com/images/avatars/lydia-deetz.jpg",
+  "location": "Winter River, CT"
+}, {
+  "first_name": "Ultimate",
+  "last_name": "Warrior",
+  "image": "http://www.radfaces.com/images/avatars/ultimate-warrior.jpg",
+  "location": "Crawfordsville, IN"
+}, {
+  "first_name": "Boba",
+  "last_name": "Fett",
+  "image": "http://www.radfaces.com/images/avatars/boba-fett.jpg",
+  "location": "Kamino"
+}, {
+  "first_name": "Garth",
+  "last_name": "Algar",
+  "image": "http://www.radfaces.com/images/avatars/garth-algar.jpg",
+  "location": "Aurora, IL"
+}, {
+  "first_name": "Jen",
+  "last_name": "",
+  "image": "http://www.radfaces.com/images/avatars/jen.jpg",
+  "location": "Thra"
+}, {
+  "first_name": "Linda",
+  "last_name": "Barret",
+  "image": "http://www.radfaces.com/images/avatars/linda-barret.jpg",
+  "location": "San Diego, CA"
+}, {
+  "first_name": "Scott",
+  "last_name": "Howard",
+  "image": "http://www.radfaces.com/images/avatars/scott-howard.jpg",
+  "location": "Beacontown"
+}, {
+  "first_name": "Shōtarō",
+  "last_name": "Kaneda",
+  "image": "http://www.radfaces.com/images/avatars/akira.jpg",
+  "location": "Neo-Tokyo, Japan"
+}, {
+  "first_name": "Dionne",
+  "last_name": "Davenport",
+  "image": "http://www.radfaces.com/images/avatars/dionne.jpg",
+  "location": "Beverly Hills, CA"
+}, {
+  "first_name": "Johnny",
+  "last_name": "Lawrence",
+  "image": "http://www.radfaces.com/images/avatars/johnny-lawrence.jpg",
+  "location": "Reseda, CA"
+}, {
+  "first_name": "Lucas",
+  "last_name": "",
+  "image": "http://www.radfaces.com/images/avatars/lucas.jpg",
+  "location": "Empire, DE"
+}, {
+  "first_name": "Winnie",
+  "last_name": "Cooper",
+  "image": "http://www.radfaces.com/images/avatars/winnie-cooper.jpg",
+  "location": "Harper Woods"
+}, {
+  "first_name": "Marian",
+  "last_name": "Delario",
+  "image": "http://www.radfaces.com/images/avatars/marian-delario.jpg",
+  "location": "Black Warriors, Turf"
+}, {
+  "first_name": "Ororo",
+  "last_name": "Munroe",
+  "image": "http://www.radfaces.com/images/avatars/ororo-munroe.jpg",
+  "location": "Harlem, NY"
+}, {
+  "first_name": "Pee-wee",
+  "last_name": "Herman",
+  "image": "http://www.radfaces.com/images/avatars/pee-wee-herman.jpg",
+  "location": "Pee-wee's Playhouse"
+}, {
+  "first_name": "Wanda",
+  "last_name": "Woodward",
+  "image": "http://www.radfaces.com/images/avatars/wanda-woodward.jpg",
+  "location": "Baltimore, MD"
+}, {
+  "first_name": "Willow",
+  "last_name": "Ufgood",
+  "image": "http://www.radfaces.com/images/avatars/willow-ufgood.jpg",
+  "location": "Tir Asleen"
+}]
```

Comment:
> but a few things to consider...
> - Typing this key will be necessary, so things like {{#each (data "radfaces")}} will be in the source.
- In some cases it might be desirable to change the contents of that file (adding new entries, for example, that may or may not be rad faces, but just people faces).
- In the context of the data directory, it's the only thing that isn't practically named

Great points @erikjung. I'm torn, honestly. I'm not sure what to do. :|